src/excmds.ts

```diff
@@ -2736,6 +2736,43 @@ export async function sleep(time_ms: number) {
     await new Promise(resolve => setTimeout(resolve, time_ms))
 }
 
+const promptPromises = {}
+const promptPrefix = `(don't edit the question): `;
+//#content
+export function triprompt(...q: string[]) {
+    const question = q.join(" ");
+    if (promptPromises[question] !== undefined) {
+        throw new Error(`Question (${question}) already asked but not answered.`);
+    }
+    return new Promise((resolve, reject) => {
+        promptPromises[question] = resolve;
+        fillcmdline_notrail(`tridactylprompt ${promptPrefix}${question}`);
+    });
+}
+
+//#content
+export function tridactylprompt(...ans: string[]) {
+    const answer = ans.join(" ").slice(promptPrefix.length);
+    const question = Object.keys(promptPromises)
+        .find(q => answer.startsWith(q))
+    if (question === undefined) {
+        throw new Error("No command waiting for this answer was found. Did you edit the question?");
+    }
+    const resolve = promptPromises[question];
+    delete promptPromises[question];
+    resolve(answer.slice(question.length));
+}
+
+//#content
+export async function tridactylprompttest() {
+    const answer = await triprompt("Do you like Tridactyl [y/n]? ");
+    if (answer === "y") {
+        fillcmdline("# Thank you!");
+    } else {
+        fillcmdline("# :(");
```

Comment:
```suggestion
        tridactylprompttest()
```

Reply:
Ha, ignore this, this was a test I didn't mean to commit.